x/concentrated-liquidity/fees_test.go

```diff
@@ -10,61 +10,134 @@ import (
 	"github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity/types"
 )
 
+// fields used to identify a fee position.
+type positionIdentifiers struct {
```

Comment:
note: this might be useful for: https://github.com/osmosis-labs/osmosis/issues/3980